site/gatsby-site/src/components/leaderboards/Leaderboard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+export const Leaderboard = ({ dataHash, leaderboard: { attribute, title }, limit }) => {
+  let sortedArray = [];
+
+  for (const item in dataHash) {
+    sortedArray.push({
+      label: dataHash[item].fieldValue,
+      value: dataHash[item].totalCount,
+      attribute,
+    });
+  }
+
+  sortedArray = sortedArray.sort((a, b) => {
```

Comment:
Does graphql have an order-by property that would allow for avoiding this sorting?

Reply:
No, it doesn't seem to have an order bu for group aggregations. I think the performance is good enough as it is. If we really want to have them already sorted in the client we could add a new mongo collection